llvm/test/CodeGen/AMDGPU/llvm.exp2.bfloat.ll

```diff
@@ -0,0 +1,240 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
Can you give this a bf16 suffix instead of bfloat, and add the other targets where it's not legal